themes/default/content/resources/cncf-live-building-your-dream-cloud-native-platform/index.md

```diff
@@ -0,0 +1,75 @@
+---
+# Name of the webinar.
+title: "Cloud Native Live: Build Your Dream Cloud Native Platform"
+meta_desc: "Join Matt Stratton as he live codes everything you need to get up and running on Kubernetes, such as monitoring, package management, and tracing."
+
+# A featured webinar will display first in the list.
+featured: false
+
+# If the video is pre-recorded or live.
+pre_recorded: false
+
+# If the video is part of the PulumiTV series. Setting this value to true will list the video in the "PulumiTV" section.
+pulumi_tv: false
+
+# The preview image will be shown on the list page.
+preview_image: ""
+
+# Webinars with unlisted as true will not be shown on the webinar list
+unlisted: false
+
+# Gated webinars will have a registration form and the user will need
+# to fill out the form before viewing.
+gated: false
+
+# The layout of the landing page.
+type: webinars
+
+# External webinars will link to an external page instead of a webinar
+# landing/registration page. If the webinar is external you will need
+# set the 'block_external_search_index' flag to true so Google does not index
+# the webinar page created.
+external: true
+block_external_search_index: true
+
+# The url slug for the webinar landing page. If this is an external
+# webinar, use the external URL as the value here.
+url_slug: "https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cloud-native-live-use-your-favorite-programming-language-to-build-your-dream-cloud-native-platform/"
+
+# The content of the hero section.
+hero:
+    # The title text in the hero. This also serves as the pages H1.
+    title: "Cloud Native Live: Use your favorite programming language to build your dream cloud native platform"
+    # The image the appears on the right hand side of the hero.
+    image: "/icons/containers.svg"
+
+# Webinar pages support multiple session via the 'multiple' property.
+# multiple:
+#   - datetime: 2020-02-05T10:00:00-07:00
+#     hubspot_form_id: ""
+#     gotowebinar_key: ""
+
+# Content for the left hand side section of the page.
+main:
+    # Webinar title.
+    title: "Cloud Native Live: Use your favorite programming language to build your dream cloud native platform"
+    # URL for embedding a URL for ungated webinars.
+    youtube_url: ""
+    # Sortable date. The datetime Hugo will use to sort the webinars in date order.
+    sortable_date: 2021-06-09T07:45:00-08:00
+    # Duration of the webinar.
+    duration: "2 hours"
+    # Datetime of the webinar.
+    datetime: ""
+    # Description of the webinar.
+    description: ""
+
+    # The webinar presenters
+    presenters:
+        - name: ""
+          role: ""
+
+    # A bullet point list containing what the user will learn during the webinar.
+    learn:
+        - ""
```

Comment:
Should this be empty? May not matter, but figured I'd point it out just in case.

Reply:
Ah yeah it is external so the `main` values don't need to filled in. I usually leave the section in there in case we end up with a recording and then I fill everything in when if we list the recording.